bitnami/redis/templates/configmap-scripts.yaml

```
@@ -159,6 +159,9 @@ data:
 
         replace_in_file "/opt/bitnami/redis-sentinel/etc/sentinel.conf" "^#*\s*${key} .*" "${key} ${value}" false
     }
+    sentinel_conf_add() {
+        (echo; echo "$@") >> "/opt/bitnami/redis-sentinel/etc/sentinel.conf"
```

```suggestion
        echo "\n$@" >> "/opt/bitnami/redis-sentinel/etc/sentinel.conf"
```

Would that create a subshell to add the line to the file? Not sure if that could cause issues. I would this in this way.

---

"\n$@" would output literal backslash and n. Adding -e to enable interpretation would also apply to possible escape sequences in parameters. Commited and pushed yet another alternative.